src/views/page_not_found.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2018 Ultimaker B.V.
+import * as React from 'react'
+
+// components
+import Page from '../components/page'
+import Tile from '../components/tile'
+
+// utils
+import { I18n } from '../utils/i18n'
+
+export interface PageNotFoundViewProps {
+    location: Location
+}
+
+export interface PageNotFoundViewState {
+}
+
+/**
+ * The page not found view.
+ */
+export default class PageNotFoundView extends React.Component<PageNotFoundViewProps, PageNotFoundViewState> {
```

Comment:
Maybe we should look into showing an error here (response_error.tsx) as we want users to report this because it shouldn't happen in the first place.